src/tox_lsr/test_scripts/runqemu.py

```diff
@@ -333,6 +333,8 @@ def make_setup_yml(
         else:
             setup_plays.extend(image["setup"])
     if remove_cloud_init:
+        if not setup_plays:
+            setup_plays.append(setup_play)
```

Comment:
Maybe, I'm a bit confused... Could the same `setup_play` be "append"ed twice to `setup_plays` by the line 337 and line 471?
https://github.com/richm/tox-lsr/blob/fix-setup-plays/src/tox_lsr/test_scripts/runqemu.py#L469-L471

Similar question on the line 378, too...

Reply:
> Maybe, I'm a bit confused... Could the same `setup_play` be "append"ed twice to `setup_plays` by the line 337 and line 471? https://github.com/richm/tox-lsr/blob/fix-setup-plays/src/tox_lsr/test_scripts/runqemu.py#L469-L471
> 
> Similar question on the line 378, too...

Good catch. I now have a simpler fix.
If `setup_plays` is `True` (i.e. not empty), then it will not contain `setup_play`.  So we just need to check if either `setup_plays` is non-empty, or `setup_play["tasks"]` is non-empty.  And, if `setup_play["tasks"]` is non-empty, we can append `setup_play` to `setup_plays`, because we know that `setup_plays` does not contain `setup_play`.